samples/bluetooth/direct_test_mode/src/dtm.c

```diff
@@ -213,14 +315,17 @@ static struct dtm_instance {
 	/* Enable or disable the workaround for Errata 172. */
 	bool anomaly_172_wa_enabled;
```

Comment:
We are mixing anomaly and errata for 172. If you have time I think it would be good to clean that up.
It should be errata

Reply:
I`m not sure about changing it to errata it seems like we often use errata or anomaly term in our code. At this moment, maybe I wouldn't change that yet.